adapters/Vungle/VungleAdapter/GADMediationAdapterVungle.m

```diff
@@ -146,6 +150,13 @@ - (void)loadRewardedInterstitialAdForAdConfiguration:
   [self loadRewardedAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
+- (void)loadBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguration *)adConfiguration
+                   completionHandler:(nonnull GADMediationBannerLoadCompletionHandler)completionHandler {
+  _bannerAd = [[GADMediationVungleBanner alloc] initWithAdConfiguration:adConfiguration
```

Comment:
Shouldn't we set COPPA value here as well?
```
[[GADMAdapterVungleRouter sharedInstance] setCOPPAStatus:adConfiguration.childDirectedTreatment];
```

Reply:
Added.